esrally/mechanic/telemetry.py

```diff
@@ -312,8 +312,8 @@ def __init__(self, telemetry_params, clients, metrics_store):
             for cluster_name in self.indices_per_cluster.keys():
                 if cluster_name not in clients.keys():
```

Comment:
it should be sufficient to check with `if cluster_name not in clients`.

Reply:
Fixed in https://github.com/elastic/rally/pull/505/commits/7be877663d6adde8d9a1e35bfb9eb83002a15274